docs/client_advanced.rst

```diff
@@ -451,6 +451,23 @@ In order to specify the nameservers to when resolving the hostnames,
     conn = aiohttp.TCPConnector(resolver=resolver)
 
 
+Changing source interface
+^^^^^^^^^^^^^^^^^^^
```

Comment:
The underline length must match the title text length.

Reply:
My bad, first time writing docs in this RST format. Will fix that